CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to Bicep
+We are very happy to accept community contributions to Bicep, whether those are [Pull Requests](#pull-requests), [Example Files](#example-files), [Feature Suggestions](#feature-suggestions) or [Bug Reports](#bug-reports)! Please note that by participating in this project, you agree to abide by the [Code of Conduct](./CODE_OF_CONDUCT.md), as well as the terms of the [CLA](#cla).
+
+## Getting Started
+* If you haven't already, you will need [dotnet](https://dotnet.microsoft.com/download) and [node + npm](https://nodejs.org/en/download/) installed locally to build and run this project.
```

Comment:
Do they need both or can they get away with just dotnet if they only want to work on the compiler for example?

Reply:
They can if they only work the c# bits without ever testing in VS code or playground. ALthough we probably shouldn't complicate the instructions too much and just tell them to get a fully setup dev environment.

Reply:
Agree, it's very easy to install both, so I don't see the need to complicate the instructions with setting up a partial environment.